idaes/core/util/misc.py

```diff
@@ -29,6 +29,22 @@
 _log = idaeslog.getLogger(__name__)
 
 
+# Author: Andrew Lee
+def get_default_solver():
+    """
+    Tries to set-up the default solver for testing, and returns None if not
+    available
+    """
+    if pyo.SolverFactory('ipopt').available(exception_flag=False):
+        solver = pyo.SolverFactory('ipopt')
+        solver.options = {'tol': 1e-6,
+                          'linear_solver': 'ma27'}
```

Comment:
If you are using Ipopt and ma27 is available, I believe ma27 is the default linear solver.  So the IDAES version should default to ma27.  Specifying this option causes the solve to fail if ma27 isn't available, which I guess could be what you are after.  If someone has Ipopt and no ma27 it is probably a small number of failures if you don't specify this option vs everything failing if you do.